triggers.cwt

```diff
@@ -2385,6 +2396,14 @@ alias[trigger:distance] = {
     distance = int
 }
 
+# TODO: Doc it up better
+## scope = { character province }
+### Determines the minimum distance from the scope. If used with NOT, it becomes maximum distance. Distances are pixels within the main map images & positions.txt
+alias[trigger:distance] = {
+    where = enum[provinces]
```

Comment:
This can just be merged into the previous trigger:distance right?
where = enum[provinces]
where = scope[province]

Reply:
Cool will do.